vignettes/covidHubUtils-overview.Rmd

```diff
@@ -206,68 +329,129 @@ By default `plot_forecasts()` uses JHU CSSE data as the "Observed Data" in the a
 
-The inputs to the `scored_forecasts()` include a dataframe created using `load_forecasts()` and a dataframe created using `truth_data()`. 
+The inputs to the `scored_forecasts()` include a `forecasts` data frame created by `load_forecasts()` and a `truth` data frame created by `load_truth()`. 
 
-The following code creates a dataframe of scored data based on the forecasts and truth data loaded earlier in this vignette in long format. 
+The following code scores forecasts by the corresponding truth data loaded earlier in this vignette in long format. This is the example for the US Forecast Hub:
 ```{r}
-scores <- score_forecasts(forecasts = forecasts,
-  return_format = "long",
-  truth = truth_data)
+inc_case_targets <- paste(1:4, "wk ahead inc case")
+
+truth_data <- load_truth(truth_source = "JHU",
+                         target_variable = "inc death",
+                         locations = "US")
+
+forecasts_multiple <- load_forecasts(models = c("COVIDhub-baseline", "COVIDhub-ensemble"),
+                      forecast_dates = as.Date(c("2020-12-14", "2020-12-15")) + seq(0, 35, 7),
+                      locations = "US",
+                      types = c("point","quantile"),
+                      targets = paste(1:4, "wk ahead inc death"),
+                      source = "zoltar",
+                      verbose = FALSE,
+                      as_of=NULL,
+                      hub = c("US"))
+
+scores <- score_forecasts(forecasts = forecasts_multiple,
+                          return_format = "wide",
+                          truth = truth_data)
+
+datatable(scores, 
+          extensions = 'FixedColumns',
+          options = list(dom = 't', scrollX = TRUE,
+                         fixedColumns = list(leftColumns = 2)))
+
 ```
 
-```{r, echo=FALSE}
-datatable(scores,
-  extensions = 'FixedColumns',
-  options = list(
-  dom = 't',
-  scrollX = TRUE,
-  fixedColumns = list(leftColumns = 2)
-))
+
+This is the example for European Forecast Hub:
+```{r}
+
+truth <- load_truth("JHU",
+                    hub = c("ECDC","US"),    
+                    target_variable = "inc death", 
+                    locations = "GB")
+
+scores_ECDC <- score_forecasts(forecasts = forecasts_ECDC,
+                               return_format = "wide",
+                               truth = truth)
+
+datatable(scores_ECDC,
+          extensions = 'FixedColumns',
+          options = list(dom = 't', scrollX = TRUE,
+                         fixedColumns = list(leftColumns = 2)))
 ```
 
+The scores could be used to compare the accuracy and precision of forecasts across models, locations, horizons, and submission weeks. 
+
+You can access the [evaluation paper](https://www.medrxiv.org/content/10.1101/2021.02.03.21250974v1) to read an in-depth explanation about the methodologies and you can also visit the [evaluation reports page](https://covid19forecasthub.org/eval-reports/). 
```

Comment:
I suggest moving this up to the top of this section (maybe the second paragraph after the overview), and also adding a link to the Forecast Evaluation Dashboard built by the CMU Delphi team and the COVID-19 Forecast Hub: https://delphi.cmu.edu/forecast-eval/